.gitignore

```diff
@@ -30,5 +30,4 @@ quicktests/overlaying/list_of_quicktests.json
 *.project
 
 # Visual Studio Code
-.settings/
-
+.vscode/
```

Comment:
Also noting that vsc will change the name of the file automatically at the upgrade step.